composer.json

```diff
@@ -20,15 +20,24 @@
             "homepage": "https://github.com/friendsofsymfony/FOSHttpCache/contributors"
         }
     ],
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://github.com/ddeboer/guzzle6-adapter"
```

Comment:
Should be removed when the adapter has been [merged](https://github.com/php-http/guzzle6-adapter/pull/1).